lib/chef/provisioning/convergence_strategy/precreate_chef_objects.rb

```diff
@@ -226,10 +238,43 @@ def client_rb_content(chef_server_url, node_name)
             https_proxy #{convergence_options[:bootstrap_proxy].inspect}
           EOM
         end
+        if convergence_options[:policy_name]
+          content << <<-EOM
+            # Policyfile Settings:
+            use_policyfile true
+            policy_document_native_api true
+            policy_group "#{convergence_options[:policy_group]}"
+            policy_name "#{convergence_options[:policy_name]}"
+          EOM
```

Comment:
You'll want to either remove the leading whitespace from these, or do something like this:

```ruby
content <<-EOM.gsub(/^\s+/, '')
```

Because the whitespace will be inserted in the rendered configuration.